docs/integrations/lightgbm.rst

```diff
@@ -38,70 +79,66 @@ Simply pass ``neptune_monitor`` to the callbacks argument of ``lgb.train``
 
     gbm = lgb.train(params,
             lgb_train,
-            num_boost_round=500,
-            valid_sets=[lgb_train, lgb_eval],
-            valid_names=['train','valid'],
-            callbacks=[neptune_monitor()], # Just add this callback
+            num_boost_round = 500,
+            valid_sets = [lgb_train, lgb_eval],
+            valid_names = ['train','valid'],
+            callbacks = [neptune_monitor()], # Just add this callback
            )
 
-Monitor your lightGBM training in Neptune
------------------------------------------
-Now you can watch your lightGBM training in Neptune!
-
-Check out this |example experiment|.
+Step 4: Monitor your lightGBM training in Neptune
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Now you can switch to the Neptune tab which you had opened previously to watch the training live!
 
 .. image:: ../_static/images/integrations/lightgbm_neptuneml.png
    :target: ../_static/images/integrations/lightgbm_neptuneml.png
    :alt: lightGBM neptune.ai integration
+	   
+|Run on Colab|
 
-Full script
------------
+.. External Links
```

Comment:
Could you please add a section "More options"
where we show how to:
- log hardware
- log hyperparameters
- log model weights
- log interactive charts (ROC curve) like here https://docs.neptune.ai/integrations/tensorboard.html#log-interactive-charts

As shown here https://docs.neptune.ai/integrations/pytorch.html#pytorch-advanced-options

Also, sections How to ask for help and Other... you may like are missing.
You can just add the section from here https://docs.neptune.ai/integrations/tensorflow-keras.html